calcom/cal.com
# fix: Zoho Bigin App update (API version V1 to V2, bug fix, additional information added to events)

## What does this PR do?

1. Updated the Zoho Bigin API version (V1 -> V2)
2. Previously the app was creating contact for all the attendees which caused nameless contacts for the Guest attendees. Updated the app to create contact only for the main attendee
3. While creating contact and event, now it will include a tag `Calcom`. It will help the user to distinguish the contacts and events created from Cal.com
4. Previously, the app was only adding Additional notes and Locations to Bigin events. Now it will include Guests list, Meeting agenda, user created questions, additional notes, and Reschedule Reason.
5. Added the required environment variable to .env.appStore.example

#### Bug Fix

1. If a contact is already available in Zoho Bigin then if the same contact book an event on Cal.com, it does not add an event to Bigin.
2. Location field was not syncing although the app is sending it to Zoho Bigin.

These 2 bugs are fixed in this pull request

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- Chore (refactoring code, technical debt, workflow improvements)
- New feature (non-breaking change which adds functionality)

## How should this be tested?

- Are there environment variables that should be set?
Two environment variable needs to be set in .env.appStore `ZOHO_BIGIN_CLIENT_ID ZOHO_BIGIN_CLIENT_SECRET`

## Mandatory Tasks
